### PR TITLE
fix: Remove other/s from organization card

### LINKED ISSUE #246: Remove other/s from organization card
image

### PR DESCRIPTION
**Pull request by rajarshisamaddar**
closes #246 

Removing other/s from the organization card has been solved in this PR.

This PR resolves issue #246 

Testing instructions:
1. Please check the organization/s card where +2 others have been replaced with only +2.

### CODE REVIEW

**Review by florian-sabonchi — APPROVED** [review]

**Comment by florian-sabonchi** [comment]
Bildschirmfoto vom 2023-04-25 18-20-29
seems to be already there

**Comment by rajarshisamaddar** [comment]
> Bildschirmfoto vom 2023-04-25 18-20-29 seems to be already there

Yes it was already there but I implemented it for the ICU, Radiology .... Section

**Comment by DasProffi** [comment]
@rajarshisamaddar please always add the `closes #246` at the beginning of your PR so github automatically closes the issue after the PR is merged

This will also link PR and issue making it easier to track them

**Comment by rajarshisamaddar** [comment]
> @rajarshisamaddar please always add the at the beginning of your PR so github automatically closes the issue after the PR is merged`closes #246`
> 
> This will also link PR and issue making it easier to track them

Okay I'll keep this in mind from now onwards

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Is there any reason behind this string interpolation? I guess we can get rid of this.

**Comment by DasProffi** [comment]
> Is there any reason behind this string interpolation? I guess we can get rid of this.

there is a + and a space, I think the interpolation is fine

**Comment by MaxSchaefer** [comment]
> > Is there any reason behind this string interpolation? I guess we can get rid of this.
> 
> there is a + and a space, I think the interpolation is fine

@FoseFx what speaks against the following?
```jsx
<span>+ {notDisplayedProfiles}</span>
```

**Comment by DasProffi** [comment]
> > > Is there any reason behind this string interpolation? I guess we can get rid of this.
> > 
> > 
> > there is a + and a space, I think the interpolation is fine
> 
> @DasProffi what speaks against the following?
> 
> ```js
> <span>+ {notDisplayedProfiles}</span>
> ```

Not much, the interpolation would allow easier modifiability if we want to change it later on, but it doesn't really matter what we choose, therefore I don't see the reason for this being a blocking change request.

File: tasks/components/AvatarGroup.tsx [hunk 1]
```diff
@@ -55,7 +55,7 @@ export const AvatarGroup = ({
         notDisplayedProfiles > 0 && (
           <div
             className={tx('truncate ml-1')}>
-            <span>{`+ ${(notDisplayedProfiles)} ${notDisplayedProfiles > 1 ? translation.others : translation.other}`}</span>
+            <span>{`+ ${(notDisplayedProfiles)}`}</span>
```
Comment (MaxSchaefer) [inline tasks/components/AvatarGroup.tsx:58]:
```suggestion
            <span>+ {notDisplayedProfiles}</span>
```

File: tasks/components/cards/OrganizationCard.tsx [hunk 2]
```diff
@@ -81,7 +81,7 @@ export const OrganizationCard = ({
         {organization.wards.slice(0, maxShownWards).map(value => value.name).join(', ')}
         {notDisplayedWards > 0 && (
           <span className={tw('ml-1')}>
-            {`+ ${notDisplayedWards} ${notDisplayedWards === 1 ? translation.other : translation.others}`}
+            {`+ ${notDisplayedWards}`}
```
Comment (MaxSchaefer) [inline tasks/components/cards/OrganizationCard.tsx:84]:
```suggestion
            + {notDisplayedWards}
```
... and inline as well :)

**Review by MaxSchaefer — CHANGES_REQUESTED** [review]
Thanks! Please ensure to remove unused code. In that case the translations.

**Comment by MaxSchaefer** [comment]
Hey @rajarshisamaddar thanks for the changes. If you want to re-request the review please use the following button in GitHub. Looks great, can you still remove the already mentioned unused code? I feel confident that @florian-sabonchi can then merge this pr.
Bildschirm­foto 2023-04-26 um 8 29 35 PM

**Comment by florian-sabonchi** [comment]
Hello @rajarshisamaddar  I have removed the translations to speed up the process

**Review by MaxSchaefer — APPROVED** [review]

**Comment by use-to** [comment]
@florian-sabonchi please merge it if it's ready.

**Comment by rajarshisamaddar** [comment]
> Hello @rajarshisamaddar  I have removed the translations to speed up the process

Thankyou @florian-sabonchi

**Pull request merged**